models/experimental/falcon40b/tt/model_config.py

```diff
@@ -797,6 +797,24 @@ def get_prefill_model_config(model_config_str, input_shape, num_devices):
             fp32_dest_acc_en=True,
             packer_l1_acc=True,
         ),
+        "COMPUTE_KERNEL_HIFI4_CONFIG": ttl.tensor.WormholeComputeKernelConfig(
+            math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            math_approx_mode=True,
+            fp32_dest_acc_en=True,
+            packer_l1_acc=True,
+        ),
+        "COMPUTE_KERNEL_HIFI4_CONFIG_FP16_DEST": ttl.tensor.WormholeComputeKernelConfig(
+            math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            math_approx_mode=True,
+            fp32_dest_acc_en=False,
+            packer_l1_acc=True,
+        ),
+        "DEFAULT_MATMUL_EQUIVALENT_CONFIG": ttl.tensor.WormholeComputeKernelConfig(
```

Comment:
Is this config used anywhere? Otherwise I'd rather remove it.

Reply:
Removed